tdc/utils/load.py

```diff
@@ -302,6 +302,15 @@ def pd_load(name, path):
                                              name + "." + name2type[name]))
         elif name2type[name] == "zip":
             df = pd.read_pickle(os.path.join(path, name + "/" + name + ".pkl"))
+        elif name2type[name] == "h5ad":
+            import anndata
+            adata = anndata.read_h5ad(
+                os.path.join(path, name + "." + name2type[name]))
+            # df = pd.DataFrame(adata.X.toarray(), columns=adata.var_names, index=adata.obs_names)
+            # TODO: multi-index would help include var information in columns
```

Comment:
I think `AnnData` doesn't like multi-index tables (I'm not sure why). I was interested in a class inherited from `AnnData` but I ended up writing a wrapper around that to enable saving multi-index tables in my python object, see `ScreenPro` class [here](https://github.com/ArcInstitute/ScreenPro2/blob/master/screenpro/__init__.py)